app/herdbook.py

```diff
@@ -497,13 +497,19 @@ def update_certificate(i_number):
         data = get_certificate_data(ind, user_id)
         data.update(**request.form)
         pdf_bytes = get_certificate(data)
-        uploaded = upload_certificate(
-           pdf_bytes=pdf_bytes.getvalue(), ind_number=ind["number"]
-        )
+
+        try:
+            signed_data = sign_data(pdf_bytes)
+            uploaded = upload_certificate(
+                pdf_bytes=signed_data.getvalue(), ind_number=ind["number"]
+            )
+        except Exception as ex:  # pylint: disable=broad-except
+            print(ex)
```

Comment:
```suggestion
            APP.logger.info("Unexpected error while updating certificate "+str(ex))
```

Reply:
Yes, that's good. For now I will make them all debug lines though.